beesdoo_product/data/barcode_rule.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+	<data noupdate="1">
+		<record model="barcode.rule" id="beesdoo_product_barcode_rule">
+			<field name="name">Beescoop Product Barcodes</field>
```

Comment:
@elouan-lb you did something recently on Odoo 9 on this matters no ? does this need to be changed accordingly ?

Reply:
@remytms Elouan will show you what have been done on nine. Could you just integrate it on this PR?

Reply:
I cherry-picked the commit on migration branch (see e1720dae9106bbb30e91e9d37c454358e0df7ee4)